docs/blog/2023-03-28-timelock-on-fastnet.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Timelock Encryption is now supported mainnet"
+summary: "Following the launch of a new `fastnet` mainnet network on March 1st, 2023, we are proud to announce general availability of Timelock Encryption!"
+date: 2023-03-28
+author: Yolan Romailler, Nicolas Gailly
+tags:
+    - Features
+    - News
+    - Conferences and Events
+---
+
+The drand team has been busy during the past year working on a very exciting problem: **Timelock Encryption**. Timelock encryption enables you to encrypt a message that cannot be decrypted by anyone until a specified time in the future.
+
+You might have heard about it from us already, since we were proud to present not just one, but three new open-source projects in August 2022 bringing a new very exciting feature called “timelock encryption” to our drand testnet! 
+
+Well, wait no more: we have now finally reached “General Availability” and our *audited* Timelock Encryption scheme is **now available on a new chain running on the drand mainnet**, namely the newly launched `fastnet` network which we’ll discuss at the end of this post. 
```

Comment:
'on a new chain' could be a little confusing

Reply:
```suggestion
Well, wait no more: we have now finally reached “General Availability” and our *audited* Timelock Encryption scheme is **now compatible with the drand mainnet**, namely the newly launched `fastnet` network which we’ll discuss at the end of this post. 
```